src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import './Loader.scss';
+
+export const Loader: React.FC = () => (
+  <img
+    alt="Loader..."
+    src="https://media4.giphy.com/media/feN0YJbVs0fwA/giphy.gif?cid=ecf05e473ee41308bc611ded0ec6bd3768fe638305058e19&rid=giphy.gif"
```

Comment:
you could youse url shortener, like https://cutt.ly/
for example. this is same link after shortener https://cutt.ly/kjRAISy